lua/framework/console/console.h

```diff
@@ -24,36 +24,35 @@ extern "C"
 /**IDK why this is needed. Seems like counter doesnt work properly without it*/
```

Comment:
what do you think aout this line?

Reply:
The same think as I think about this https://github.com/Open-Sn/opensn/blob/1a63d644abd286178b92b157331854be61aff1d5/framework/object_factory.h#L10-L13

and this:
https://github.com/Open-Sn/opensn/blob/1a63d644abd286178b92b157331854be61aff1d5/framework/post_processors/post_processor_printer.cc#L15-L18

:-)

But seriously. I have not scrutinized these macros, but sometimes the preprocessor needs some help...

Reply:
Issue #249 will resolve this